main.c

```diff
@@ -5,9 +5,19 @@
 #include <unistd.h>
 #include <dlfcn.h>
 
-#ifndef __x86_64__
+/*#ifndef __x86_64__
```

Comment:
Might just wanna delete this, it’ll be in the history.

Reply:
Yeah - can you remove this ifdef for the PR